dart/tool/service_undocumented.md

```diff
@@ -158,3 +158,53 @@ class HeapSpace extends Response {
 <!-- _getCpuProfileTimeline -->
 
 <!-- _getAllocationSamples -->
+
+streamId | event types provided
```

Comment:
I'm hoping that the plan is to promote `_registerService` and related to public API once things have been proved out.